docs/README.md

```diff
@@ -650,3 +650,158 @@ You will need to distribute or share pub-cert.pem so that people can use this wi
 ### Wildcard domains with of-router
 
 Coming soon
+
+# GitLab
+
+Currently the the only way you can get your OpenFaas Cloud working with GitLab is on-prem.
```

Comment:
@ivanayov suggested to change that sentence. There is also duplicated word `the`